runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -1392,13 +1392,13 @@ ClassFileOracle::walkMethodCodeAttributeCode(U_16 methodIndex)
 	U_8 *code = codeAttribute->code;
 	for (U_32 codeIndex = 0; codeIndex < codeAttribute->codeLength; codeIndex += step) { /* NOTE codeIndex is modified below for CFR_BC_tableswitch and CFR_BC_lookupswitch */
 
-#if defined(J9VM_OPT_VALHALLA_VALUETYPES)
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
```

Comment:
please add a TODO comment saying something like:  The current edition of LWVT currently maps the ValueType bytecodes to opcodes that are used internally. This is a work around reads in the LWVT bytecodes and remaps them to opcodes that do not conflict. Once, there is an official spec that specifies which opcodes the new LWVT bytecodes will have, we will remap the internal bytecodes so they do not conflict.

Reply:
new PR: https://github.com/eclipse/openj9/pull/1234